slackbridge/utils.py

```diff
@@ -167,6 +169,9 @@ def emoji_replace(match: Match[str]) -> str:
         simple mapping above.
         """
         emoji = match.group(1)
+        unicode_emoji = emoji.emojize(f':{emoji}:', use_aliases=True)
+        if unicode_emoji != f':{emoji}:':
+            return unicode_emoji
```

Comment:
I don't think it's necessary to continue using this function-- the `emojize` function will automatically find and make all the necessary substitutions given a full block of text. So you can delete `emoji_replace` and put this in line 190 directly.